pandas/core/tools/timedeltas.py

```diff
@@ -83,8 +83,12 @@ def to_timedelta(arg, unit='ns', box=True, errors='raise'):
     elif isinstance(arg, ABCIndexClass):
         return _convert_listlike(arg, unit=unit, box=box,
                                  errors=errors, name=arg.name)
-    elif is_list_like(arg) and getattr(arg, 'ndim', 1) == 1:
-        return _convert_listlike(arg, unit=unit, box=box, errors=errors)
+    elif is_list_like(arg) and getattr(arg, 'ndim', 1) <= 1:
```

Comment:
can you separate these into 2 conditions on the main if/else, IOW

```
elif is_list_like(arg) and getattr(arg, 'ndim', 1) == 0:
     arg = arg.item()

# this would then be unchanged
elif is_list_like(arg) and getattr(arg, 'ndim', 1) == 1:
     return _convert.....
```